info.go

```diff
@@ -406,6 +410,9 @@ func (t *JSONTime) UnmarshalJSON(buf []byte) error {
 	s := bytes.Trim(buf, `"`)
 
 	if string(s) == "null" {
+		// Question: the line below is what upstream
```

Comment:
Have you figured this out? fwiw I don't think we use this method, so we should just use the upstream

Reply:
Ok, nice - that's the context I was after - will fix!